pyramid_nacl_session/__init__.py

```diff
@@ -1,12 +1,12 @@
-def _export(thing):
-    """ No-op; document that ``thing`` is an API.
-
-    Also, shuts pyflakes up about unused import.
-    """
+# -*- coding: utf-8 -*-
 
+# flake8: noqa
 from .serializer import EncryptedSerializer
-_export(EncryptedSerializer)
 from .session import EncryptedCookieSessionFactory
-_export(EncryptedCookieSessionFactory)
 from .scripts import generate_secret
-_export(generate_secret)
```

Comment:
Rather than suppressing all `flake8` checks, please add the exported names to `__all__` to appease `flake8` (rather than the `_export` hack).